### PR TITLE
f-fix/gh-1089-date-picker-initial-value

### PR DESCRIPTION
**Pull request by jprotic**
Resolves: #1089

### CODE REVIEW

File: src/main/frontend/src/pages/administration/components/years/EditYear.js [hunk 1]
```diff
@@ -65,14 +65,17 @@ class EditYear extends Component {
...
     return (
       <Formik
         validationSchema={validationSchema}
         initialValues={{
           ...this.props.year,
-          carriedOverHoursExpirationDate: moment(
-            this.props.year.carriedOverHoursExpirationDate
-          ).toDate()
+          carriedOverHoursExpirationDate:
+            carriedOverHoursExpirationDate === null
```
Comment (jolpazy) [inline src/main/frontend/src/pages/administration/components/years/EditYear.js]: will !carriedOverHoursExpirationDate work here?

Comment (jolpazy) [reply src/main/frontend/src/pages/administration/components/years/EditYear.js]: maybe even carriedOverHoursExpirationDate ||  moment(carriedOverHoursExpirationDate).toDate()

File: src/main/frontend/src/pages/administration/components/years/EditYear.js [hunk 2]
```diff
@@ -65,14 +65,17 @@ class EditYear extends Component {
...
+              ? carriedOverHoursExpirationDate
+              : moment(carriedOverHoursExpirationDate).toDate()
```
Comment (jolpazy) [inline src/main/frontend/src/pages/administration/components/years/EditYear.js]: did we make a formatter for this?

File: src/main/frontend/src/pages/administration/components/years/EditYear.js (hunk 1 above)
Comment (jprotic) [reply src/main/frontend/src/pages/administration/components/years/EditYear.js]: Only first one works.

**Pull request merged**